### DISCUSSION TITLE
Suggestion: "Frequency" option for auto-fill on Privileges page

### DISCUSSION THREAD

**Discussion by fschein** [sws2apps/organized-app#2934] [Ideas]
Hi there,

I’ve been looking at the Hourglass app, and I think some of its options on the privileges page could be interesting for Organized. Specifically, I believe the "Frequency" option for auto-fill might be a useful feature.

I’m not sure how the development of this screen is currently going, but I think it could be something to consider for the future.

![Screenshot From 2024-11-13 21-43-47](https://github.com/user-attachments/assets/f2623a00-ab8a-4ea7-b360-ad9ba44300e3)

**Comment by ux-git** [comment]
Thanks for the idea! We're aiming to make the Organized app easy to use, but something like this might make sense and adds some flexibility. We'll take a closer look. In the meantime, if/when you start using the app after its release, please feel free to share your feedback regarding the Autofill feature — this will help us make it even better.

**Reply by fschein** [reply]
I'm looking forward to starting to use it on day one. I'll definitely provide feedback. I actually want to start feeding Organized today so that when it updates to the new version, it will already be almost ready. Will the data be migrated?
Thank you for the great work!

**Reply by rhahao** [reply]
Hello @fschein, at this moment, it is not yet sure if data entered in the "Organized Preview" app will be migrated to the new "Organized". If this will be changed in the coming days, we’ll let you know.

The migration process we already set in place is for users who currently use the "CPE" app. For them, their data will be migrated once the new app is released.

We hope this information can be helpful.

**Reply by ux-git** [reply]
@fschein If you'd like to get started right now and migrate later, using the current "Congregation Program for Everyone" (CPE) version is the way to go. You can register and begin entering your congregation's data there, and it will migrate smoothly to the new version once it's released. Moving data from preview environments like preview.organized-app.com would delay the launch, so we’re focusing on making the CPE → Organized migration process as seamless as possible.

Additionally, keep in mind that the new Organized app offers a far superior user experience, so you might find it easier to wait for its release and then enter all your data quickly and effortlessly. Also, the new version includes some new fields and data options, meaning that while migration from the old CPE app is possible, some additional data entry will still be needed to make full use of the new features.

**Reply by fschein** [reply]
Hmm, okay. So I think I’ll wait to start fresh on Organized when it’s released. Would it be possible to delete my account and the data I’ve added so I can start from scratch later? I think it will be more practical to begin anew. The way it is now, it might get complicated with some entries already in the system...

**Reply by rhahao** [reply]
Affirmative @fschein. Since these data are not connected at all, so don’t worry, you can start fresh new when Organized is released publicly. Thank you.

**Reply by ux-git** [reply]
@rhahao will the data migration be optional or necessarily? I mean could @fschein simply wait until the release happens without doing the backup or using the migration window? Could you share more details?

**Reply by rhahao** [reply]
Correct @ux-git! Since he will wait till Organized is released then there is no data to be migrated for him at all. Migration will only happen for previous CPE users. Thanks.
